app/assets/javascripts/common/reports.js

```diff
@@ -438,7 +438,10 @@ DashboardReports = () => {
       return withBaseLineConfig(config);
     },
     overdueReturnToCareTrend: function (data) {
-      const monthsDashed = 2
+      const currentDate = new Date();
+      const currentDayOfTheMonth = currentDate.getDate();
```

Comment:
I think you mean `currentDateOfTheMonth`

Reply:
Corrected